tests/sentry/incidents/action_handlers/test_email.py

```diff
@@ -363,10 +400,18 @@ def test_links_customer_domains(self):
         action = self.create_alert_rule_trigger_action(triggered_for_incident=incident)
         result = generate_incident_trigger_email_context(
             self.project,
-            incident,
-            action.alert_rule_trigger,
-            trigger_status,
-            IncidentStatus(incident.status),
+            organization=incident.organization,
+            metric_issue_context=MetricIssueContext.from_legacy_models(
+                incident=incident,
+                new_status=IncidentStatus(incident.status),
+            ),
+            alert_rule_serialized_response=self.serialize_alert_rule(incident.alert_rule),
+            incident_serialized_response=self.serialize_incident(incident),
+            alert_context=AlertContext.from_alert_rule_incident(incident.alert_rule),
+            open_period_context=OpenPeriodContext.from_incident(incident),
+            trigger_status=trigger_status,
+            trigger_threshold=action.alert_rule_trigger.alert_threshold,
+            user=self.user,
```

Comment:
If we're repeating the same input logic, does it make sense to extract this out into a helper method that takes legacy models and invokes the email context util?